SofaKernel/modules/SofaDeformable/RestShapeSpringsForceField.inl

```diff
@@ -76,24 +88,21 @@ void RestShapeSpringsForceField<DataTypes>::bwdInit()
 
     if (restMState.get() == NULL)
     {
-      useRestMState = false;
+        useRestMState = false;
 
-      msg_error(this)<< "external_rest_shape in node " << this-> getContext()->getName() << " not found";
-      //getContext()->removeObject(this);
-      return;
```

Comment:
I just checked and I agree with @EulalieCoevoet. 
The return wouldn't allow a default scenario with no external_rest_shape. 
Sorry, my bad !!!